src/osd/OSD.cc

```diff
@@ -8431,7 +8431,8 @@ void OSD::check_osdmap_features()
   }
 
   if (osdmap->require_osd_release < CEPH_RELEASE_NAUTILUS) {
-    heartbeat_dispatcher.ms_set_require_authorizer(false);
+    hb_front_server_messenger->set_require_authorizer(false);
+    hb_back_server_messenger->set_require_authorizer(false);
```

Comment:
shall we set them to `true` if `require_osd_release` is true and these flags are false?